package.json

```diff
@@ -39,7 +39,7 @@
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.1001.6",
+    "@angular-devkit/build-angular": "^0.1002.0",
```

Comment:
Would it be possible to revert this change please? As well as changes of package-lock and gitignore? Thanks!

Reply:
absolutely